tox.ini

```
@@ -1,29 +1,21 @@
 [tox]
 envlist =
   docs
-  py27-sphinx{17,18}
-  py{36,37,38}-sphinx{17,18,20,21,22,23,24,30,31,32,33,34}
+  py27-sphinx1
+  py{36,37,38,39}-sphinx{2,3,latest}
```

Can we keep the test for Python3 and Sphinx 1.x?

```suggestion
  py{36,37,38,39}-sphinx{1,2,3,latest}
```

---

I think they dropped support for 3.8 on 2.x or something, but it should run just fine.